src/front-door/azext_front_door/_params.py

```diff
@@ -65,7 +65,8 @@ def load_arguments(self, _):
     with self.argument_context('network front-door', arg_group='HTTPS') as c:
         c.argument('certificate_source', arg_type=get_enum_type(FrontDoorCertificateSource), help='Certificate source to enable HTTPS.')
         c.argument('secret_name', help='The name of the Key Vault secret representing the full certificate PFX')
-        c.argument('secret_version', help='The version of the Key Vault secret representing the full certificate PFX')
+        c.argument('secret_version', help='The version of the Key Vault secret representing the full certificate PFX, '
```

Comment:
Are there any other versions supported besides `Latest`?

Reply:
Yes, I've added two test: One for Latest version and one for specific version.

For the time being, those tests will fail as our existing enable-https implementation has ignored the LROPoller which will cause failures in TearDown, I've marked them as skipped.

Another way to work around this is we could wait for the operations to finish using az network frontend-endpoint wait, but that only works in live_only model, it seems that VCR.py does not work well with the ignored underlying LRO poll, and each test will take more than 1 hour to finish.